addons/beehave/nodes/beehave_tree.gd

```diff
@@ -8,3 +9,12 @@ enum {
 	FAILURE,
 	RUNNING
 }
+
+
+func _get_configuration_warnings() -> PackedStringArray:
+	var warnings: PackedStringArray = []
+	
+	if get_children().any(func(x): return not (x is BeehaveNode)):
```

Comment:
This would not give a warning if `get_children()` is empty, right? We need to cover that case, too.

Reply:
Yeah, but I think it depends on Beehave node type. For example Leaf nodes **should** have no children, while composites should have at least 2 and decorators exactly 1 etc., so I implemented children count check separately depending on script. This line just checks if all children have proper type (that is BeehaveNode, otherwise it appends a warning).